content/en/blog/trust-in-your-wallet/index.md

```diff
@@ -0,0 +1,150 @@
+
+## The Cryptographic Magic
+
+The core concept of self-sovereign identity is verified data. The idea is that entities can hold
+cryptographically signed data (aka credentials) in their digital identity wallets. They can create
+proofs of this data and hand it over to other parties for verification.
+
+This process is what happened in the demo mentioned above: the user made proof of his credential
+and handed it over to the identity provider. The identity provider then verified the proof
+cryptographically and read and provided the needed data to the OIDC client application.
+
+{{< figure src="/docs/slides/building-findy-agency/trust-triangle.png" width="900px"
+attr="<small>The trust triangle describes the roles of the data verification scenario.</small>"
+attrlink="/docs/slides/building-findy-agency/trust-triangle.png" >}}
+
+The cryptographic magic during the verification allows the verifier to be sure of two aspects.
+Firstly, the issuer was the one who issued the credential, and secondly, the issuer issued
+it to the prover's wallet (and no one else's). The verifier must know the issuer's
+public key for this magic to happen.
+
+## Who to Trust?
+
+So cryptography gets our back regarding the authenticity of the data and its origin. But how can we
+trust the issuer, the entity that created the credential initially?
+
+{{< figure src="https://i.imgflip.com/2ab5u1.jpg" width="800px">}}
+
+The answer to this question is no different from the present situation regarding dealing with third
+parties. *We choose to trust the issuer.*
+
+For example, consider the current OIDC flow, where identity providers deliver data from their silos
+to the client applications. The client application implementers consciously decide to trust that
+the IdP runs their server in a documented URL and that the IdP provides them with correct data in
+the OIDC protocol flows. They may have secrets and keys to keep the OIDC data exchange secure and
+private, but ultimately, the decision to trust the OIDC provider is part of the application design.
+
+In the SSI scenario, we choose to trust the issuer similarly, only we are not aware of the issuer's
+servers but their public key for signing the credential. In our OIDC login demo,
+the client application ("issuer-tool") has decided to trust the "FTN service" that has issued
+the data to the user's wallet.
+
+## Finnish Trust Network
+
+[Finnish Trust Network](https://www.kyberturvallisuuskeskus.fi/en/our-activities/regulation-and-supervision/electronic-identification)
+(FTN) consists of "strong electronic identification" providers. The concept
+means proving one's identity in electronic services that meets specific requirements laid down
+by Finnish law. The identity providers are required to implement multi-factor authentication
+to identify the users. The result data of the authentication process typically contains
+the user's name and personal identification code. Often users authenticate using their bank ID.
```

Comment:
Last paragraph.. should be say: usually users use their banks as IdPs or something. When I think how difficult those who don't know Finnish system (TUPAS) is to understand this part, I think it needs clarification.